osutil/sys/syscall.go

```diff
@@ -20,7 +20,9 @@
 package sys
```

Comment:
maybe call the file "runas.go" or something? syscall seems overly generic for this :)

Reply:
Nice suggestion, did this